pkg/server/server.go

```diff
@@ -72,5 +74,5 @@ func (s *ReplicationServer) Shutdown() {
 }
 
 func parsePrivateKey(privateKeyString string) (*ecdsa.PrivateKey, error) {
-	return crypto.HexToECDSA(privateKeyString)
+	return crypto.HexToECDSA(strings.TrimPrefix(privateKeyString, "0x"))
```

Comment:
Lets us re-use the PK environment variable we are using with forge to deploy the contract locally, which expects a `0x` prefix. HexToECDSA does not expect the `0x`